packages/next/src/server/image-optimizer.ts

```diff
@@ -240,6 +241,10 @@ export class ImageOptimizerCache {
       }
     }
 
+    if (INVALID_URL_REGEX.test(url)) {
+      return { errorMessage: '"url" parameter is invalid' }
```

Comment:
```suggestion
      return { errorMessage: '"url" parameter cannot start with backslash' }
```

Let's be more specific here so we can confirm the new tests are hitting this condition.